src/person.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'nameable'
+require_relative 'decorator'
+# Public: Various methods useful for performing checking on people in a library.
+class Person < Nameable
+  # Public: Gets/Sets the String name or age of the person.
+  attr_accessor :name, :age

```

Comment:
- [ ] Kindly add the getter-setter for the `:rentals` property here

Reply:
resolved 👍